cmd/install.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/packwiz/packwiz/core"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var installCmd = &cobra.Command{
+	Use:     "install [name] [url]",
+	Short:   "Install a mod from a url this is different from modrinth/curseforge install",
```

Comment:
"Add an external file from a direct download link, for sites that are not directly supported by packwiz" maybe? this is very vague and informal